scripts/websocket_client.py

```diff
@@ -0,0 +1,59 @@
+#! /bin/env python
+
+from websocket import WebSocket
+from os import environ
+import time
+
+import requests
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+
+FORUM_HOST = environ.get("FORUM_HOST", "127.0.0.1")
+FORUM_PORT = environ.get("FORUM_PORT", 8000)
+FORUM_USER_EMAIL = environ.get("FORUM_USER_EMAIL")
+FORUM_USER_PASSWORD = environ.get("FORUM_USER_PASSWORD")
+
+
+def get_jwt_access() -> str:
+    response = requests.post(
+        f"http://{FORUM_HOST}:{FORUM_PORT}/users/token/",
+        json={
+            "email": FORUM_USER_EMAIL,
+            "password": FORUM_USER_PASSWORD
+        }
+    )
+    if response.status_code != 200:
+        print("[-] Password auth failed")
+        exit(0)
+
+    access_token = response.json().get("access")
+    if not access_token:
+        print("[-] There is no access token in response JSON")
+        exit(0)
+
+    return access_token
```

Comment:
Use logging instead of `print` statements. Add error handling for potential issues.